package.json

```diff
@@ -12,7 +12,8 @@
     "dev": "turbo run dev --parallel --filter='!./templates/*'",
     "test": "NODE_OPTIONS=--no-warnings npx vitest",
     "lint": "turbo run lint",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\""
+    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "superflare": "./packages/superflare/bin/superflare.js"
```

Comment:
FYI added this so you can `pnpm superflare <command>` more easily without the finicky behavior of `npx`. Also: `./scratch` exists as a locally ignored subfolder for purposes of scaffolding 👍